olm-catalog/serverless-operator/index/configs/index.yaml

```diff
@@ -3,34 +3,34 @@ schema: olm.channel
 name: stable
 package: serverless-operator
 entries:
-  - name: "serverless-operator.v1.28.0"
   - name: "serverless-operator.v1.29.0"
-    replaces: "serverless-operator.v1.28.0"
-    skipRange: ">=1.28.0 <1.29.0"
   - name: "serverless-operator.v1.30.0"
     replaces: "serverless-operator.v1.29.0"
     skipRange: ">=1.29.0 <1.30.0"
+  - name: "serverless-operator.v1.30.1"
+    replaces: "serverless-operator.v1.30.0"
+    skipRange: ">=1.30.0 <1.30.1"
 ---
 schema: olm.channel
 name: stable-1.29
 package: serverless-operator
 entries:
-  - name: "serverless-operator.v1.28.0"
   - name: "serverless-operator.v1.29.0"
-    replaces: "serverless-operator.v1.28.0"
-    skipRange: ">=1.28.0 <1.29.0"
   - name: "serverless-operator.v1.30.0"
     replaces: "serverless-operator.v1.29.0"
     skipRange: ">=1.29.0 <1.30.0"
+  - name: "serverless-operator.v1.30.1"
```

Comment:
Just a note for the future because we have been setting it wrong for this stable-1.29 channel. It should include versions up to 1.29.x. Not 1.30 or newer.

Reply:
I've created a new issue from your comment to have captured outside of the PR. But I assume it's doesn't break anything yet.